src/server/acl/helpers.cc

```diff
@@ -346,4 +353,16 @@ std::string AclKeysToString(const AclKeys& keys) {
   return result;
 }
 
+std::string PasswordsToString(const absl::flat_hash_set<std::string>& passwords, bool nopass,
+                              bool full_sha) {
+  if (nopass) {
+    return "nopass ";
+  }
+  std::string result;
+  for (const auto& pass : passwords) {
+    absl::StrAppend(&result, "#", PrettyPrintSha(pass, full_sha), " ");
+  }
+
+  return result;
+}
```

Comment:
so it can very well be that a user doesn't have nopass set, but it's password set is empty. Maybe we should print some kind of warning in that case?

Reply:
I am following redis/valkey here :) If there is no `nopass` or anything that starts with `#` it means that there is no password set and the field is left blank